### ISSUE TITLE
Components with missing inverse definition

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#948]
### Definition

Revisar el caso inverse de los siguientes componentes:

- [x] #1655
- [x] #736
- [x] #1788
- [ ] #1786
- [x] #1936

**Comment by yceballost** [comment]
Uh, bien cazado ese inverse del spinner

**Comment by aweell** [comment]
O2 ha levantado la necesidad de tener una correcta visualización de las chips sobre inverse, he añadido el ejemplo en el cuerpo de la issue, por el momento van a hacer la propuesta inverse de ese ejemplo hasta que exista solución en mística.

**Comment by sandrafernandezsimon** [comment]
buenas! hay estimación sobre esto?

**Comment by yceballost** [comment]
No! esto siempre fue una necesidades que vimos desde hace tiempo pero en ningún equipo se ha pedido. Si lo necesitáis metedlo en los okrs de q2 y así tenemos la excusa de darle salida a esto :)
